pom.xml

```diff
@@ -28,10 +28,12 @@
 	</modules>
 
 	<properties>
-		<openMRSVersion>2.0.3</openMRSVersion>
+		<openMRSVersion>2.1.6</openMRSVersion>
+		<bahmniVersion>0.94-SNAPSHOT</bahmniVersion>
```

Comment:
Why not use 0.94.1 for java utils. See also comment on the vairable name above

Reply:
@angshu, We modified version to 0.94.1